src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | undefined,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: undefined,
+  };
+
+  componentDidMount() {
+    window.addEventListener('keyup', this.keyListener);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('keyup', this.keyListener);
+  }
+
+  keyListener = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: event.key,
```

Comment:
try using `event.code`, to check if it is `space` pressed, cause now you have this:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/55802630/182556863-71110dde-232b-4697-8125-d1f8a6d1c481.png">